share/spice/skyscanner_flight_search/footer.handlebars

```diff
@@ -0,0 +1,8 @@
+<div class="fix">
```

Comment:
@samheyman I think you'll want `oneline` here.

On mobile these aren't remaining aligned to the bottom